$OpenBSD: patch-configure_ac,v 1.1 2017/10/29 15:17:36 ajacoutot Exp $

XXX cyclic: lang/vala -> math/graphviz -> x11/gnome/librsvg -> lang/vala

Index: configure.ac
--- configure.ac.orig
+++ configure.ac
@@ -98,36 +98,6 @@ PKG_CHECK_MODULES(GMODULE, gmodule-2.0 >= $GLIB_REQUIR
 AC_SUBST(GMODULE_CFLAGS)
 AC_SUBST(GMODULE_LIBS)
 
-PKG_CHECK_MODULES(LIBGVC, libgvc >= $LIBGVC_REQUIRED)
-AC_MSG_CHECKING([for CGRAPH])
-cgraph_tmp_LIBADD="$LIBADD"
-cgraph_tmp_CFLAGS="$CFLAGS"
-LIBADD="$LIBADD $LIBGVC_LIBS"
-CFLAGS="$CFLAGS $LIBGVC_CFLAGS"
-AC_RUN_IFELSE(
-	[AC_LANG_SOURCE([
-		#include <gvc.h>
-
-		int main(void) {
-			#ifdef WITH_CGRAPH
-				return 0;
-			#else
-				return -1;
-			#endif
-		}
-	])], [
-		AC_MSG_RESULT([yes])
-		VALAFLAGS="$VALAFLAGS -D WITH_CGRAPH"
-		have_cgraph=yes
-	], [
-		AC_MSG_RESULT([no])
-		have_cgraph=no
-	]
-)
-LIBADD="$cgraph_tmp_LIBADD"
-CFLAGS="$cgraph_tmp_CFLAGS"
-AM_CONDITIONAL(HAVE_CGRAPH, test "$have_cgraph" = "yes")
-
 AC_PATH_PROG([XSLTPROC], [xsltproc], :)
 AM_CONDITIONAL(HAVE_XSLTPROC, test "$XSLTPROC" != :)
 
@@ -156,16 +126,6 @@ AC_CONFIG_FILES([Makefile
            vapigen/Makefile
            vapigen/vala-gen-introspect/Makefile
            vapigen/vala-gen-introspect/vala-gen-introspect
-           libvaladoc/valadoc.pc
-           libvaladoc/valadoc.deps
-           libvaladoc/Makefile
-           valadoc/Makefile
-           valadoc/doclets/Makefile
-           valadoc/doclets/html/Makefile
-           valadoc/doclets/devhelp/Makefile
-           valadoc/doclets/gtkdoc/Makefile
-           valadoc/icons/Makefile
-           valadoc/tests/Makefile
 ])
 
 AC_OUTPUT
